src/core/lib/event_engine/posix_engine/posix_endpoint.cc

```diff
@@ -100,6 +109,89 @@ ssize_t TcpSend(int fd, const struct msghdr* msg, int* saved_errno,
 }
 
 #ifdef GRPC_LINUX_ERRQUEUE
+
+#define GRPC_LOG_EVERY_N_SEC(n, format, ...)                                  \
```

Comment:
I'd recommend putting this with general logging utils as well. And is time_precise required here?

Reply:
Moved it to src/core/lib/gprpp/time.h